src/pages/components/menu/Menu.js

```diff
@@ -0,0 +1,97 @@
+//Libraries
+import React from 'react';
+import PropTypes from 'prop-types';
+
+//Helpers
+import saveItem from '../../../api/saveItem';
+import getUpdatedQuantityPerDishById from './helpers/getUpdatedQuantityPerDishById';
+import getUpdatedCartItems from './helpers/getUpdatedCartItems';
+
+//Constants
+import { INITIAL_STATE } from './constants/menu.general';
+
+//Components
+import MenuLinks from './components/menuLinks';
+import Dishes from './components/dishes';
+import Cart from './components/cart';
+
+//CSS
+import './menu.css';
+
+class Menu extends React.Component {
+  state = INITIAL_STATE;
+  resetCartItems() {
+    this.setState({
+      cartItems: [],
+      quantityPerDishById: {},
+    });
```

Comment:
Why are you not directly setting it to the initial state?

Reply:
If we add any other key to state then we have to change here also

Reply:
then you can store this separately as cart state and use that here